test/study-webidl.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Tests the Web IDL analysis library.
+ */
+
+const assert = require('assert').strict;
+const { studyWebIdl } = require('../src/lib/study-webidl');
+
+describe('The Web IDL analyser', async _ => {
+  const specUrl = 'https://www.w3.org/TR/spec';
+
+  function toCrawlResult(idl) {
+    return [
+      {
+        url: specUrl,
+        title: 'Sample spec',
+        crawled: specUrl,
+        shortname: 'spec',
+        nightly: { repository: 'https://github.com/w3c/spec' },
+        idl
+      }
+    ];
+  }
+
+  it('reports no anomaly if IDL is valid', async () => {
+    const crawlResult = toCrawlResult(
+`[Global=Window,Exposed=*]
+interface valid {};`);
```

Comment:
```suggestion
interface Valid {};`);
```
At some point I assume we'll check naming conventions that require interfaces to be capitalized, so we may as well use a valid name :)